docs-content/general/6_product-features/4_logging/1_overview.md

```diff
@@ -6,7 +6,7 @@ slug: overview
 
 You can find our logging product at [app.highlight.io/logs](https://app.highlight.io/logs). 
 
-If you're language of choice isn't support in the "Getting Started" docs below, hit us up in [our community](https://highlight.io/community) or send us an email.
+If your language of choice isn't support in the "Getting Started" docs below, hit us up in [our community](https://highlight.io/community) or send us an email at support@highlight.io
```

Comment:
Added a support email to the end here. I did file #5071 so we can figure out how to handle support links long term.

Reply:
Thanks for validating!